code/formfields/UserFormsNumericField.php

```diff
@@ -57,7 +57,11 @@ public function validate($validator)
         if ((!empty($this->min) && $this->value < $this->min) || (!empty($this->max) && $this->value > $this->max)) {
             $validator->validationError(
                 $this->name,
-                sprintf("This must be between %s and %s", $this->min, $this->max),
+                _t(
+                    'UserFormsNumericField.ErrorMsg',
```

Comment:
Thanks for making this change, it's great! However could you please update the key here to be consistent with [others](https://github.com/silverstripe/silverstripe-userforms/blob/master/lang/en_GB.yml#L2) present in the module

e.g. `self::class . '.RANGE_ERROR'` or something along these lines in the format of:
1. the class name to begin
2. a dot
3. then a capital case identifier

Also, have you considered what might happen if the user has specified a minimum, but not a maximum (or vice versa)?

Reply:
@NightJar i've updated the error message. please, let me know what you think.